uPortal-webapp/src/main/resources/properties/security.properties

```diff
@@ -38,14 +38,14 @@ portal.context=/uPortal
 portal.protocol.server.context=${portal.protocol}://_CURRENT_SERVER_NAME_${portal.context}
 portal.login.url=${portal.protocol.server.context}/Login
 
-# Needed for proxy-cas in load-balanced system like apache to give back the proxy-ticket on the server that requested it
-# it requires that the load-balanced instance has his serverName.
+# Needed for proxy-cas in load-balanced system like apache to give back the
+# proxy-ticket on the server that requested it it requires that the
```

Comment:
Ponctuation is needed: ``` . It requires```

Reply:
Else sorry that you encountered a problem with my commit, and thanks to fix it !
Now I have to purpose a fix on the CAS client to trim white-spaces !